src/main/java/MassimoBoi.java

```diff
@@ -0,0 +1,128 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+public class MassimoBoi {
+    public static void main(String[] args) {
+
+        String logo = " __  __               _                   _           _ \n" +
+                "|  \\/  | __ _ ___ ___(_)_ __ ___   ___   | |__   ___ (_)\n" +
+                "| |\\/| |/ _` / __/ __| | '_ ` _ \\ / _ \\  | '_ \\ / _ \\| |\n" +
+                "| |  | | (_| \\__ \\__ \\ | | | | | | (_) | | |_) | (_) | |\n" +
+                "|_|  |_|\\__,_|___/___/_|_| |_| |_|\\___/  |_.__/ \\___/|_|";
+
+        System.out.println("Hello from\n" + logo);
+        Scanner myObj = new Scanner(System.in);
+        String userInput = "";
+        List<Task>taskList = new ArrayList<>();
+        greet();
+
+        /*
+         * The while loop takes in user input and processes them according to different user inputs
+         * List of user inputs: task type, mark and unmark task, list, bye (to end chat)
+         */
+        while(true){
+            userInput = myObj.nextLine();
+            if(userInput.equals("bye")){
+                goodbye();
+                break;
+            }
+            handleInput(userInput, taskList);
+
+        }
+
+    }
+
+    public static void handleInput(String userInput, List<Task>taskList){
+         if(userInput.equals("list")){
+            printList(taskList);
+            makeHorizontalRow();
+            return;
+        } else if(userInput.contains("unmark")){
+            String[] handleInput = userInput.split(" ");
+            int taskToUnmark = Integer.parseInt(handleInput[1]) - 1;
+            taskList.get(taskToUnmark).unmark();
+            System.out.printf("%s unmarked. Type 'list' to see updated list\n Type mark [list index] to mark this task\n"
+                    ,taskList.get(taskToUnmark).getDescription());
+            makeHorizontalRow();
+            return;
+        } else if(userInput.contains("mark")){
```

Comment:
can also use ".startsWith("mark")" since the ip commands all start with the command words